openedx/core/djangoapps/user_api/completion/tasks.py

```diff
@@ -0,0 +1,192 @@
+"""Module containing a task for user progress migration"""
+
+
+import logging
+import unicodecsv
+
+from io import BytesIO
+
+from celery.task import task
+from completion.models import BlockCompletion
+from completion_aggregator.models import Aggregator
+from courseware.courses import get_course
```

Comment:
this should be absolute path -> `from lms.djangoapps.courseware.courses import get_course`